sphinxcontrib/wavedrom.py

```diff
@@ -127,16 +129,39 @@ def determine_format(supported):
     return None
 
 
+def render_wavedrom_cli(code):
+    source = NamedTemporaryFile(mode="w", delete=False)
+    output = NamedTemporaryFile(delete=False)
+    source.write(code)
+    # Windows can't use the name a second time while the first file is open
+    source.close()
+    output.close()
+    args = ["npx", "wavedrom-cli", "-i", source.name, "-s", output.name]
+    try:
+        subprocess.run(args, check=True)
+    except subprocess.CalledProcessError:
+        raise  # TODO: handle case when unable to find executable
+    with open(output.name, "r") as svg:
+        svg_text = svg.read()
+    del source, output
+    return svg_text
+
+
 def render_wavedrom(self, node, outpath, bname, format):
     """
     Render a wavedrom image
     """
 
     # Try to convert node, raise error with code on failure
-    try:
-        svgout = render(node["code"])
-    except JSONDecodeError as e:
-        raise SphinxError("Cannot render the following json code: \n{} \n\nError: {}".format(node['code'], e))
+    if True:
```

Comment:
I didn't implement a flag yet. For my purposes, I just forced it to always use the cli.